pyme-depends/meta.yaml

```diff
@@ -128,15 +128,18 @@ requirements:
   run:
     - python {{ python }}
     - numpy >=1.11
+    - numpy >= 1.4 [py3k]
     - scipy
     - matplotlib
     - wxpython
     #<3.9 [py2k] #FIXME - there are still some issues with display logic on wx=4.x
     #- wxpython-phoenix < 3.9 [py3k]
     #- wxpython [py3k] #FIXME - there are still some issues with display logic on wx=4.x
+    - wxpython <= 4.0.4 [py3k]

```

Comment:
Is this exclusive to py3? And do we really want to pin?

Reply:
Again, thought you'd be more receptive if we just changed this for py3. We definitely need to pin. Any version higher and nothing works, and it's been that way for months. The default packages always install a newer version of wxpython and the default install is always busted until I downgrade.

Reply:
Just for the record, could you clarify what you mean by 'nothing works' - is it just traits, or does it effect other stuff too.

Reply:
By nothing works I mean I can't launch `pymevisualize`, `dh5view`, or `pymeacquire`. I get an error before the program even comes up.